python/foxglove-sdk/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@foxglove/notebook",
+  "version": "0.0.0",
+  "license": "MIT",
+  "author": {
+    "name": "Foxglove Technologies",
+    "email": "support@foxglove.dev"
+  },
+  "homepage": "https://foxglove.dev/",
+  "scripts": {
+    "dev": "yarn run build --sourcemap=inline --watch",
+    "build": "esbuild ts/widget.ts --format=esm --bundle --outdir=python/foxglove/notebook/static",
+    "build:prod": "yarn build --minify",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@foxglove/embed": "0.1.2"
+  },
+  "devDependencies": {
+    "@anywidget/types": "0.2.0",
+    "esbuild": "0.25.9",
+    "typescript": "5.9.2"
+  }
```

Comment:
probably all should be devDependencies as this is not a library package, rather it just produces an output bundle with no runtime dependencies if I understand correctly?